src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -799,7 +800,7 @@ private function getLinkObject(string $resourceClass, string $operationId, strin
             return [];
         }
         $linkObject['operationId'] = $operationId;
-        $linkObject['description'] = 1 === \count($identifiers) ? sprintf('The `%1$s` value returned in the response can be used as the `%1$s` parameter in `GET %2$s`.', $identifiers[0], $path) : sprintf('The values returned in the response can be used in `GET %s`.', $path);
+        $linkObject['description'] = 1 === \count($identifiers) ? sprintf('The `%1$s` value returned in the response can be used as the `id` parameter in `GET %2$s`.', $identifiers[0], $path) : sprintf('The values returned in the response can be used in `GET %s`.', $path);
```

Comment:
I hardcoded the `id` parameter name, as it is hardcoded in OperationPathResolver.php in line 52:

```
        if (OperationType::ITEM === $operationType) {
            $path .= '/{id}';
        }
```

So even if you display the right property name, the parameter in the path is always shown as {id}.

I think a new issue should be created to create the url identifier parameter name dynamically. It is a complex change as it could affect to all the Path Resolvers implemented.

Reply:
`{id}` is a URI template variable, so its name doesn't matter, does it?

Reply:
Hi @dunglas ,

If you don't replace the second `%1$s` placeholder with the harcoded `id` string, and your identifier property is not called id, Swagger UI links section will display something like this if you don't set a custom path or route that operation: 

`The **code** value returned in the response can be used as the **code** parameter in GET /api/issues/{**id**}.`

Both placeholders will be replaced with the identifier property name. So the user is looking for a `code` parameter in the URL, but there isn't any, as it is always display as `/resource-path/{id}` in the URL.

But if you change the second placeholder to a fixed `id` string, anyone setting a custom path (for example `/api/issues/{code}`) or route name in the operation would have to use id as their Resource identifier parameter, or will display something like this:

`The **code** value returned in the response can be used as the **id** parameter in GET /api/issues/{**code**}.`

So if you call your property identifier id or you don't use a custom path or route name in the operation, the two options are fine. For most users, I think the first option will show the right combination, as it is aligned with the default API Plaform behaviour. I think there are more users changing their identifier property name than setting custom paths or routes.

You can see the current behaviour in this screenshot:
![Screenshot_2020-10-29 API Platform](https://user-images.githubusercontent.com/1867132/97613236-bac5e200-1a18-11eb-85a2-06006dde4930.png)

Reply:
```suggestion
         $linkObject['description'] = 1 === \count($identifiers) ? sprintf('The `%1$s` value returned in the response can be used as the `%1$s` parameter in `GET %2$s`.', $identifiers[0], $path) : sprintf('The values returned in the response can be used in `GET %s`.', $path);
```